billing.rb

```diff
@@ -11,6 +11,8 @@
 ORG_GUID = '386a9502-d9b6-4aba-b3c3-ebe4fa3f963e'
 API_URL = 'https://api.london.cloud.service.gov.uk'
 PRECISION = 2
+SERVICE_CHARGE = 10.0 / 100
+BILLING_API_OPENING_HOURS = { open: 8, close: 24 }.freeze
```

Comment:
You could use a range here:

```ruby
BILLING_API_OPENING_HOURS = (8..24).freeze
```

And then you can use `BILLING_API_OPENING_HOURS.begin` and `BILLING_API_OPENING_HOURS.end`, as well as `BILLING_API_OPENING_HOURS.include?(now.hour)`